client/src/utils/timeline-event/getClassNames.js

```diff
@@ -3,23 +3,24 @@ const getTimelineEventClassNames = (formattedEvent) => {
     isPast,
     isCurrent,
     isConfirmed,
+    isPastAndConfirmed,
+    isClosed,
     hasMissingMaterials,
   } = formattedEvent;
 
   const classNames = ['timeline-event'];
 
-  if (isPast) {
-    classNames.push('timeline-event--past');
-  }
-
-  if (isCurrent) {
-    classNames.push('timeline-event--current');
-  }
-
-  if (isConfirmed) {
-    classNames.push('timeline-event--locked');
-
-    if (!isPast) {
+  if (isPastAndConfirmed) { // Then event can be closed
+    classNames.push(isClosed ? 'timeline-event--closed' : 'timeline-event--unclosed');
+  } else { // event either not past or not isConfirmed
```

Comment:
Idem, commentaire sur la ligne précédente :    
_(Même si je pense que le code parle de lui même)_

```suggestion

  // Event either not past or not isConfirmed
  } else {
```